rpcs3/rpcs3qt/debugger_frame.h

```diff
@@ -49,12 +51,13 @@ class debugger_frame : public custom_dock_widget
 
 	u64 m_threads_created = -1;
 	u64 m_threads_deleted = -1;
+	system_state m_emu_state{};
 	u32 m_last_pc = -1;
 	std::vector<char> m_last_query_state;
 	u32 m_last_step_over_breakpoint = -1;
 
-	std::shared_ptr<CPUDisAsm> m_disasm;
-	std::weak_ptr<cpu_thread> cpu;
+	std::shared_ptr<CPUDisAsm> m_disasm; // Only shared to allow base/derived functionality
+	std::shared_ptr<cpu_thread> cpu;
```

Comment:
Might be a good time to change this to m_cpu

Reply:
Done.